packages/fxa-content-server/server/bin/fxa-content-server.js

```diff
@@ -87,6 +87,7 @@ logger.info('page_template_directory: %s', PAGE_TEMPLATE_DIRECTORY);
 function makeApp() {
   const app = express();
   const settingsPath = '/settings';
+  const betaContentServerPath = '/beta';
```

Comment:
I know we talked about nixing the `/beta` and just going with `/` for routes for users in the experiment so the URL doesn't change, was that not possible?